relay-server/src/actors/processor.rs

```diff
@@ -1761,6 +1820,100 @@ fn encode_payload(body: &Bytes, http_encoding: HttpEncoding) -> Result<Bytes, st
     Ok(envelope_body.into())
 }
 
+/// Serializes the given Envelope and applies the configured HTTP encoding.
+///
+/// This function overrides the `sent_at` timestamp stored in the envelope. Since the envelope went
+/// through basic normalization, all timestamps have been corrected. We propagate the new `sent_at`
+/// to allow the next Relay to double-check this timestamp and potentially apply correction again.
+/// This is done as close to sending as possible so that we avoid internal delays.
+fn encode_envelope(
```

Comment:
Should we inline this instead? Would make the modification of the envelope's `sent_at` more obvious.

Reply:
Implemented in 13f4955, what do you think?